glances/plugins/plugin/model.py

```diff
@@ -330,14 +330,14 @@ def get_stats_snmp(self, bulk=False, snmp_oid=None):
         ret = {}
         if bulk:
             # Bulk request
-            snmp_result = snmp_client.getbulk_by_oid(0, 10, *list(itervalues(snmp_oid)))
+            snmp_result = snmp_client.getbulk_by_oid(0, 10, *list(.values(snmp_oid)))
             logger.info(snmp_result)
             if len(snmp_oid) == 1:
                 # Bulk command for only one OID
                 # Note: key is the item indexed but the OID result
                 for item in snmp_result:
-                    if iterkeys(item)[0].startswith(itervalues(snmp_oid)[0]):
-                        ret[iterkeys(snmp_oid)[0] + iterkeys(item)[0].split(itervalues(snmp_oid)[0])[1]] = itervalues(
+                    if .keys(item)[0].startswith(.values(snmp_oid)[0]):
```

Comment:
As well; the methods [`keys()`](https://docs.python.org/3/library/stdtypes.html#dict.keys) and [`values()`](https://docs.python.org/3/library/stdtypes.html#dict.values) should be used.

Reply:
Typo